This patch contains the device settings proto changes that were introduced in
the chromium repo per crrev.com/c/2908073. We unfortunately can't roll
protofiles right now due Authpolicy tests failure after picking up the latest
proto changes (see crbug.com/1234804). This patch is a temporary stop gap
solution to allow work that depends on the proto changes to land. Once protofiles
can be properly upreved again, the patch is no longer needed and should be dropped.

--- a/cloud/policy/proto/chrome_device_policy.proto
+++ b/cloud/policy/proto/chrome_device_policy.proto
@@ -1759,11 +1759,19 @@
 }
 
 // Setting that controls whether data access is enabled for Thunderbolt/USB4
-// peripherals.
+// peripherals. This proto is no longer being used, please use
+// DevicePciPeripheralDataAccessEnabledProtoV2.
 message DevicePciPeripheralDataAccessEnabledProto {
   optional bool enabled = 1 [default = false];
 }
 
+// Setting that controls whether data access is enabled for Thunderbolt/USB4
+// peripherals. This replaces DevicePciPeripheralDataAccessEnabledProto. Used
+// only for the associated CrosSetting.
+message DevicePciPeripheralDataAccessEnabledProtoV2 {
+  optional bool enabled = 1;
+}
+
 // Setting that controls whether Borealis will be allowed on the device.
 message DeviceBorealisAllowedProto {
   optional bool allowed = 1 [default = true];
@@ -1946,4 +1954,6 @@
   optional DeviceDebugPacketCaptureAllowedProto
       device_debug_packet_capture_allowed = 121;
+  optional DevicePciPeripheralDataAccessEnabledProtoV2
+      device_pci_peripheral_data_access_enabled_v2 = 123;
 }
